### DISCUSSION TITLE
failed to parse proxy DWHVBSEJ, err: local ip or plugin is required

### DISCUSSION THREAD

**Discussion by mxthd** [natfrp/wiki#307] [使用问题]
mac docker 需要使用docker.for.mac.localhost访问宿主机，在后台将ip设置为docker.for.mac.localhost后，启动docker出现错误：
failed to parse proxy DWHVBSEJ, err: local ip or plugin is required

**Comment by fakeboboliu** [comment]
这个问题的修复已经被推到各镜像 registry，请重新 pull 使用。

给您带来的不便十分抱歉。

**Reply by mxthd** [reply]
> 这个问题的修复已经被推到各镜像注册表，请重新拉使用。
> 
> 给您带来的不方便的十分抱住。

hello, 重新拉了latest版，上面的问题已经解决，可以正常启动，但是好像无法联通宿主机，报错：
`
2023/04/12 11:39:22 [E] [76/989969/6634] [iim**veq.DWHVBSEJ] 连接映射目标 [docker.for.mac.localhost:8080] 失败, 请检查本地服务是否可访问: dial tcp: lookup docker.for.mac.localhost on 100.115.92.200:53: no such host`

同样的docker.for.mac.localhost:8080我配置到nginx镜像是可以正常代理的

**Reply by fakeboboliu** [reply]
> 同样的docker.for.mac.localhost:8080我配置到nginx镜像是可以正常代理的

`docker.for.mac.localhost` 已经不存在于 docker 文档中，试试 `host.docker.internal` 吧。
